### PR TITLE
Comment class

### PR DESCRIPTION
**Pull request by ghost**
Added Like/unlike and counting likes functions

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -15,11 +15,12 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
```
Comment (amshali) [inline .project:15]: Add this file to your .gitignore file.

File: app/models/Comment.java [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
 import java.util.*;
 import javax.persistence.*;
+
+import controllers.Secure;
  
 import play.db.jpa.*;
  
 @Entity
-public class Comment extends Model {
+public class Comment extends Commentable {
```
Comment (amshali) [inline app/models/Comment.java:11]: Why is this subclassed from Commentable? Can someone comment on a comment?

File: app/models/Likes.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package models;
+
+import java.util.*;
+import javax.persistence.*;
+
+import controllers.Secure;
+import play.db.jpa.*;
+ 
+@Entity
+public class Likes extends Model{
```
Comment (amshali) [inline app/models/Likes.java:10]: Check out my post in piazza regarding likeable

File: bin [hunk 4]
```diff
@@ -0,0 +1 @@
+Subproject commit 097f386f4e19b387caa0ee49884b819cc43d1291
```
Comment (amshali) [inline bin:1]: Please remove this from commit. Add this folder (or file) to .gitignore file in your project

File: app/models/Comment.java (hunk 2 above)
Comment (robertleeshook) [inline app/models/Comment.java:11]: i think Commentable carries some factored out code that will work for anything that is a comment

Comment (amshali) [inline app/models/Comment.java:11]: But non one can comment on a comment. This should not inherit from Commentable i think.

Comment (ghost) [inline app/models/Comment.java:11]: Oh I totally forgot about that. Will fix that soon

Comment (amshali) [inline app/models/Comment.java:11]: drop me a note after you fixed it.

Comment (amshali) [inline app/models/Comment.java:11]: did you fix this?

Comment (ghost) [inline app/models/Comment.java:11]: Hi Amin,

I'm sorry about the delay. I had a test today. There's a little bug I'm trying to find. I will get this fix before tomorrow.

Sincerely,

Di Huynh
Computer Science and Mathematics
The University of Texas at Austin

**Pull request closed without merging**